generators/client/templates/angular/src/main/webapp/app/_app.module.ts

```diff
@@ -68,7 +58,10 @@ let routerConfig = {
         { provide: Window, useValue: window },
         { provide: Document, useValue: document },
         customHttpProvider(),
-        PaginationConfig
+        PaginationConfig,
+        JhiLanguageService,
```

Comment:
This service is already provided by the `ng-jhipster` lib so it shouldnt be provided again here. it will cause issues